scripts/generate_parser_test_files.py

```diff
@@ -5,16 +5,24 @@
 
 from tqdm import tqdm
 
-from fundus import Crawler, PublisherCollection
+from fundus import BaseCrawler, Crawler, PublisherCollection
 from fundus.logging import basic_logger
 from fundus.publishers.base_objects import PublisherEnum
 from fundus.scraping.article import Article
+from fundus.scraping.html import FundusSource
+from fundus.scraping.scraper import Scraper
 from tests.test_parser import attributes_required_to_cover
 from tests.utility import HTMLTestFile, get_test_case_json, load_html_test_file_mapping
 
 
-def get_test_article(enum: PublisherEnum) -> Optional[Article]:
-    crawler = Crawler(enum)
+def get_test_article(enum: PublisherEnum, url: Optional[str] = None) -> Optional[Article]:
```

Comment:
This is not critical for now but `url` shadows the name from the outer scope. It would be nice anyway to wrap the main script into a main function to avoid global exposure.

Reply:
Good point, i restructured the script a bit.